extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -61,18 +63,19 @@ public static function onOasisSkinAssetGroups( &$jsAssets ) {
 	/**
 	 * Return whether we're on one of the pages where we want to show the Recirculation widgets,
 	 * specifically File pages, Article pages, and Main pages
+	 *
 	 * @return bool
 	 */
 	static public function isCorrectPageType() {
 		$wg = F::app()->wg;
 
-		$showableNameSpaces = array_merge( $wg->ContentNamespaces, [ NS_FILE ] );
+		$showableNameSpaces = array_merge( $wg->ContentNamespaces, [ NS_FILE, NS_BLOG_ARTICLE ] );
 
 		if ( $wg->Title->exists() &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)